segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java

```diff
@@ -355,6 +363,7 @@ private ServiceConfig(TypedProperties properties) throws ConfigurationException
         this.cachePolicy = new CachePolicy(cachePolicyMaxSize, cachePolicyTargetUtilization, cachePolicyMaxUtilization,
                 Duration.ofSeconds(cachePolicyMaxTime), Duration.ofSeconds(cachePolicyGenerationTime));
         this.replyWithStackTraceOnError = properties.getBoolean(REPLY_WITH_STACK_TRACE_ON_ERROR);
+        this.requestProcessorMaxReadLength = properties.getInt(REQUEST_PROCESSOR_MAX_READ_LENGTH);
```

Comment:
Validate that it is positive. And optionally greater than some min value. Let's make it consistent with PravegaRequestProcessor/Listener